src/main/java/com/conveyal/datatools/manager/utils/MergeFeedUtils.java

```diff
@@ -0,0 +1,225 @@
+package com.conveyal.datatools.manager.utils;
```

Comment:
This file and really every other file related to feed merging would be better organized within a package dedicated to feed merging classes in the jobs package.

Reply:
Handled in https://github.com/ibi-group/datatools-server/pull/376/commits/0ec01886c326fe6881c9c22a4025120ad505580f.